src/content/docs/it/getting-started.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Per Iniziare
+description: Un'introduzione base ad Astro
```

Comment:
di Astro

Reply:
Given that it's an introduction to Astro's concepts I think "ad Astro" is more accurate here. It's like the difference between "introduction to" and "introduction of".

https://www.treccani.it/vocabolario/introduzione/